resources/lib/sosac.py

```diff
@@ -353,9 +362,56 @@ def list_tv_recently_added(self, url):
                 result += self.list_tv_recently_added(url)
         return result
 
+    def library_movies_all_xml(self):
+        page = util.request('http://tv.prehraj.me/filmyxml.php')
+        pagedata = util.substr(page, '<select name=\"rok\">', '</select>')
```

Comment:
Teď si nejsem jistý zda je to copy-paste již existující fce library_tvshows_all_xml nebo library_movie_recently_added_xml plus lehce upravený kód, aby dělal co má.
Pokud to tedy není dobře, pak už původní kód musí být "špatně".